showcase/src/app/home/home.component.html

```diff
@@ -1,100 +1,114 @@
-<h1 class="mat-display-2" translate>Stark documentation</h1>
+<div class="about-background">
+	<h1 class="about-title" [innerHTML]= "'SHOWCASE.HOMEPAGE.MAIN_TITLE' | translate"></h1>
+</div>
+<h2 class="mat-body-2" [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_BEFORE_DETAIL' | translate">
+</h2>
+<h4 [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_DETAIL' | translate"></h4>
+<h2 class="mat-body-2" [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_AFTER_DETAIL' | translate">
```

Comment:
2 things: 
- Can you use a `<p>` instead of `<h2>`? Because this is not a title, just normal text
- And can you remove the `mat-body-2` class to not have the bold text? otherwise it is strange that some text is in bold and some other is not.